Python/Python.sublime-syntax

```diff
@@ -3,8 +3,9 @@
 name: Python
 file_extensions:
   - py
-  - rpy
   - pyw
+  - pyi
+  - rpy
```

Comment:
PythonImproved also has `py3`, would you want to add that? Was there ever an equivalent `py2`?